rom/dev/README.md

```diff
@@ -668,6 +668,42 @@ Following is the sequence of steps that are performed to download the firmware i
 
 See Firmware [Image Validation Process](#firmware-image-validation-process).
 
+### Derivation of the hash chain for Stable Identity
```

Comment:
I think it would be more accurately called a KDF ratchet, or a KDF chain.

Reply:
Yes, I've been thinking I should rename that. Changed to "key ladder".